### PR TITLE
Go update 1.19 to 1.22

### PR DESCRIPTION
**Pull request by karl-johan-grahn**

### CODE REVIEW

File: .github/workflows/pull_request.yaml
Comment (karl-johan-grahn) [inline .github/workflows/pull_request.yaml:4]: Temporarily set to `pull_request` to be able to test the new version of golangci-lint in the PR workflow, need to change this back after approval and before merge if pipeline is successful

**Review by tanalam2411 — DISMISSED** [review]

**Comment by karl-johan-grahn** [comment]
Will force merge since workflow has been verified to work when running using `pull_request`

**Pull request merged**
